app/design/adminhtml/default/default/template/bundle/sales/creditmemo/create/items/renderer.phtml

```diff
@@ -9,13 +9,14 @@
  * @category    design
  * @package     default_default
  * @copyright   Copyright (c) 2006-2020 Magento, Inc. (https://www.magento.com)
- * @copyright   Copyright (c) 2018-2022 The OpenMage Contributors (https://www.openmage.org)
+ * @copyright   Copyright (c) 2018-2024 The OpenMage Contributors (https://www.openmage.org)
  * @license     https://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
  */
 ?>
 <?php
 /**
  * @see Mage_Bundle_Block_Adminhtml_Sales_Order_Items_Renderer
```

Comment:
Possible to remove line 18?

Reply:
see annotation is in lot of files and its used for phpdocumentor. I'd keep.

Reply:
How about consistency? The `see` annotation is also not in a lot of files. For consistency, is it better to remove?

Reply:
"See" can have different or multiple values.

I am not going to add them when not helpfull, but i'd keep the existing.